QuizGame/README.md

```diff
@@ -0,0 +1,28 @@
+# About project "QuizGame":
+## This is a simple version of quiz written on Python that connects with database.
+
+# How to start. First steps
+    git clone https://github.com/siriusdevs/rpm-hws_7-2_2023
+
+    git checkout kulgavykh
+
+    cd quizgame
+
+# Next step is creating Docker container and filling database
+    run init_script.sh
+
+### Then, you shoul fill the .env file
+# .env
+    HOST - database host
```

Comment:
how do i know this creds? you set docker pg up yourself with your script.
maybe you could set up .env file in that script too?